calculate.rb

```diff
@@ -0,0 +1,58 @@
+class Calculate
+	attr_accessor :calc_choice, :nums, :operator, :sum
+
+	def initialize
+	end
+
+	def do_calculation
+		@@nums.map! {|i| Integer(i)}
+	
+		if @@operator == "1"
+			sum = @@nums.inject(0){|sum,x| sum + x }
+			puts sum
```

Comment:
This is a bit abrupt and not very user friendly. I would at least have a message saying something like "The answer is #{sum}".